data/pathfinder/paizo/roleplaying_game/ultimate_magic/um_templates.lst

```diff
@@ -35,3 +35,5 @@ Starting Gold UM ~ Maximum	VISIBLE:NO																																	KIT:1|Star
 Starting Gold UM ~ Average	VISIBLE:NO																																	KIT:1|Starting Gold UM ~ Average
 
 Magus Spellbook			VISIBLE:NO																																	KIT:1|Magus Spellbook
+
+Mindchemist Language	VISIBLE:NO			LANGBONUS:ALL
```

Comment:
Unless they can learn secret languages, you may want to use the standard method of "TYPE=Spoken"